inspirehep/modules/records/api.py

```diff
@@ -71,6 +74,13 @@ def delete(self):
     def _delete(self, *args, **kwargs):
         super(InspireRecord, self).delete(*args, **kwargs)
 
+    def commit(self, validator=InspireValidator):
```

Comment:
A possible case would be on writing some tests and we don't want to have mock records that will validate through these custom rules. Otherwise, it is decided to be used as the default commit method. We can discuss about it.